packages/react-components/src/templates/ContactInfoModal.tsx

```diff
@@ -126,7 +126,7 @@ const ContactInfoModal: React.FC<ContactInfoModalProps> = ({
             <LabeledTextField
               title="Website 2"
               subtitle="(Optional)"
-              pattern={USER_SOCIAL_WEBSITE.source}
```

Comment:
Should this be tested?

Reply:
<img width="710" alt="image" src="https://user-images.githubusercontent.com/9073485/167439239-665c15fc-503c-471d-9b01-b69f97483078.png">